src/main/java/org/example/Enum/BedroomsQuantityOptions.java

```diff
@@ -0,0 +1,16 @@
+package org.example.Enum;
```

Comment:
**suggestion** - package name should be all lowercase, i.e. `Enum` -> `enum`. I would even suggest we keep the naming convention you have, so maybe it should be `enums`

Reply:
name changed!